<!--
    File: index.html
    Project: reveal.js
    Created: 2021-11-09 13:30:32
    Author: Hyunbin Kim (khb7840@gmail.com)
    Description:
		This is the main file for the reveal.js presentation.
		This includes the 3rd presentation for microbiome lecture.
	---
    Last Modified: 2021-11-16 11:19:55
    Modified By: Hyunbin Kim (khb7840@gmail.com)
    ---
    Copyright © 2021 Hyunbin Kim, All rights reserved
-->
<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>Microbiome Presentation - Hyunbin Kim</title>

		<link rel="stylesheet" href="dist/reset.css">
		<link rel="stylesheet" href="dist/reveal.css">
		<link rel="stylesheet" href="dist/theme/custom-theme.css" id="theme">

		<!-- Theme used for syntax highlighted code -->
		<link rel="stylesheet" href="plugin/highlight/monokai.css">
	</head>
	<body>
		<div class="reveal">
			<div class="slides">
				<section>
					<!-- Cover for Review paper -->
					<br><br>
					<h6>Review paper</h6>
					<h2> Interactions of commensal and pathogenic microorganisms with the intestinal <span>mucosal barrier</span></h2>
					<h4>Nat Rev Microbiol, 2018</h4>
					<p>2021-35389 Hyunbin Kim / 2021. 11. 16</p>
				</section>

				<section data-background="img/mucosa.gif">
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Abstract</h2>
					<p>
						<div class="container" style="display:flex"><div class="col" style="flex:1">
						<div id="abs">
						<ul>
							<li>
								The intestinal mucosal barrier is composed of <span>epithelial cells</span> that are protected by an <span>overlying host-secreted mucous layer</span> and functions as <span>the first line of defence</span> against pathogenic and non-pathogenic microorganisms.
							</li>
							<li>
								Some microorganisms have <span>evolved strategies</span> to either <span>survive in the mucosal barrier</span> or circumvent it to establish infection.
							</li>
						</ul>
						<img id="01c" src="img/Review01c.jpeg" alt="01c" width=80%>
						<h6>https://doi.org/10.1038/d41586-018-07277-1</h6>
						</div>
						</div>
						<div class="col">
							<img id="01a" src="img/Review01a.png" alt="01a" width="60%">
						</div>
					</div>
				</section>

				<!--Explanation on figure 01a-->
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<div class="container" style="display:flex"><div class="col" style="flex:1">
					<img id="01a" src="img/Review01a.png" alt="01a" width="100%">
					</div>
					<div class="col" style="flex:1.5">
						<h2 id="architecture">Large intestinal crypt architecture</h2>

						<h5 class="highlighted">Commensal bacteria</h5><br>
							<p>
								The large intestine harbours commensal bacteria that ferment dietary fibre into short-chain fatty acids (SCFAs), which are an important energy source for colonic enterocytes and goblet cells.
							</p>
						<img src="img/commensal.png" alt="" width=40%>
						<h5>
							10<sup>11</sup> - 10<sup>12</sup>microorganisms per cm<sup>3</sup> in colon
						</h5>
						<h5>
							Mainly <span>Bacteroidetes and Firmicutes</span>
						</h5>
					</div>
					</div>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<div class="container" style="display:flex"><div class="col" style="flex:1">
					<img id="01a" src="img/Review01a.png" alt="01a" width="100%">
					</div>
					<div class="col" style="flex:1.5">
						<h2 id="architecture">Large intestinal crypt architecture</h2>

						<h5 class="highlighted">Commensal bacteria</h5><br>
							<p>
								The large intestine harbours commensal bacteria that ferment dietary fibre into short-chain fatty acids (SCFAs), which are an important energy source for colonic enterocytes and goblet cells.
							</p>

						<h5 class="highlighted">Goblet cells</h5><br>
							<p>
								Mucus secreted by goblet cells continually replenishes the mucous layer that overlies the gut epithelium as a first barrier against commensal bacteria and invading pathogens.
							</p>
						<img src="img/goblet.png" alt="">
						<h4>
							Gut epithelial cells are connected by tight junctions.
						</h4>
					</div>
					</div>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<div class="container" style="display:flex"><div class="col" style="flex:1">
					<img id="01a" src="img/Review01a.png" alt="01a" width="100%">
					</div>
					<div class="col" style="flex:1.5">
						<h2 id="architecture">Large intestinal crypt architecture</h2>
						<h5 class="highlighted">Commensal bacteria</h5><br>
							<p>
								The large intestine harbours commensal bacteria that ferment dietary fibre into short-chain fatty acids (SCFAs), which are an important energy source for colonic enterocytes and goblet cells.
							</p>

						<h5 class="highlighted">Goblet cells</h5><br>
							<p>
								Mucus secreted by goblet cells continually replenishes the mucous layer that overlies the gut epithelium as a first barrier against commensal bacteria and invading pathogens.
							</p>

						<h5 class="highlighted">IgA & AMPs</h5>
							<p>
								Secretory immunoglobulin A (IgA) and antimicrobial peptides (AMPs) are secreted into the mucus as a defence against pathogens and potentially harmful commensal bacteria.
							</p>
						<img src="img/iga_amp.png" alt="">
						</div>
					</div>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">

					<div class="container" style="display:flex"><div class="col" style="flex:1">
					<img id="01a" src="img/Review01a.png" alt="01a" width="100%">
					</div>
					<div class="col" style="flex:1.5">
					<h2>The gut mucosal barrier</h2>
					<p>As epithelial cells renew every few days, the secretory activity of goblet cells continuously replenishes the mucous layer that overlies the epithelium and lubricates the intestine.</p>
					<br>
					<h4>Two mucous layers</h4>
						<ol>
							<li>Firm inner layer: adherent to the epithelium and <span>resistant to dense microbial colonization</span></li>
							<li>Loose outer layer: home to a dense population of <span>commensal bacteria</span></li>
						</ol>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">

					<div class="container" style="display:flex"><div class="col" style="flex:1">
					<img id="01a" src="img/Review01a.png" alt="01a" width="100%">
					</div>
					<div class="col" style="flex:1.5">
					<h2>The gut mucosal barrier</h2>
					<p>As epithelial cells renew every few days, the secretory activity of goblet cells continuously replenishes the mucous layer that overlies the epithelium and lubricates the intestine.</p>
					<br>
						<h4>Example of interaction between commensal bacteria and mucous layer</h4>
						<ul>
							<li><span>Bacteroides thetaiotaomicron</span> and <span>Escherichia coli</span>, exhibit different global transcriptional patterns during existence in the colonic mucous layer.</li>
							<li><span>B. thetaiotaomicron</span> and <span>Faecalibacterium prausnitzii</span> modulate mucus production by augmenting goblet cell differentiation and inducing expression of genes involved in mucin glycosylation.</li>
						</ul>
				</section>


				<!--Colonic epithelium cells -->
				<section data-auto-animate>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
					<img id="01a" src="img/cells.png" alt="" width=75%>
					</div>
					<div class="col" style="flex:2">
							<h2>The colonic epithelium consists of</h2>
							<ul>
								<li>enteroendocrine cells</li>
								<li>tuft cells</li>
								<li>cup cells</li>
								<li>goblet cells</li>
								<li>sentinel goblet cells</li>
							</ul>
							<br><br>
							<p>These cells are located at the crypt entrance to reduce bacterial encroachment into the crypt.</p>
					</div>
				</div>
				</section>

				<section data-auto-animate>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
					<img id="01a" src="img/cells.png" alt="" width=75%>
					</div>
					<div class="col" style="flex:2">
							<h2>The colonic epithelium consists of</h2>
							<ul>
								<li>enteroendocrine cells</li>
								<h6> produce and release hormones in response to a number of stimuli</h6>
								<li>tuft cells</li>
								<h6>chemosensory cells in the epithelial lining of the intestines</h6>
								<li>cup cells</li>
								<h6>have distinct cell types but functions are not well-known</h6>
								<li>goblet cells</li>
								<h6>secrete the mucus layer that protects the epithelium from the lumenal contents</h6>
								<li>sentinel goblet cells</li>
								<h6>guards the colonic crypt by triggering Nlrp6-dependent MUC2 secretion</h6>
							</ul>
							<br><br>
							<p>These cells are located at the crypt entrance to reduce bacterial encroachment into the crypt.</p>
					</div>
				</div>
				</section>

				<!--About MUC2 - Fig 01b -->
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<br>
					<h2>MUC2 glycoproteins are a major constituent of colonic mucus</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
							<img id="muc" src="img/monomer.png" alt="">
						</div>
						<div class="col" style="flex:1">
							<p id="mucin-paragraph">
								<div>
								The importance of MUC2 is underscored by the fact that mice genetically deficient in the Muc2 gene develop inflammation and colon cancer owing to closer contact of bacteria with the epithelium.<br><br>
								</div>
								Mucin monomer contains <span>PTS domains;</span><br>
								which is a region of rich Proline(<b>P</b>), Threonine(<b>T</b>), and Serine(<b>S</b>).

							</p>
						</div>
					</div>
				</section>
				<section data-auto-animate>
					<br>
					<h2>MUC2 glycoproteins are a major constituent of colonic mucus</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
							<img id="muc" src="img/monomer.png" alt="">
							<div class="r-stack" width=80%>
								<div class="fragment fade-out" data-fragment-index="0">
									<h6>https://en.wikipedia.org/wiki/O-linked_glycosylation</h6>
									<img src="img/o-glycan.png" alt="" width=50%>
								</div>
								<img class="fragment current-visible" data-fragment-index="0" id="muc" src="img/multimer.png" alt="">
							</div>
						</div>
						<div class="col" style="flex:1">
							<p id="mucin-paragraph">
								Mucin monomer contains <span>PTS domains;</span><br>
								which is a region of rich Proline(<b>P</b>), Threonine(<b>T</b>), and Serine(<b>S</b>).
								<br>
								The N-terminal and C-terminal domains mediate <span>disulfide crosslinking</span> to build a very large mucin network comprising thousands of monomers.
							<p>
								A series of <span>glycosyl-transferase enzymes</span> add thousands of individual O-linked glycans to the side chains of <b>Ser and Thr</b> residues that ultimately comprise up to <span>80% of the final MUC2 mass.</span>
							</p>
						</div>
					</div>
				</section>

				<section>
					<h2>Synthesis of mucin glycans</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
						<img src="img/Review02.webp" alt="review02" width=200%>
						</div>
						<div class="col" style="flex:1.5">
							<ul>
								<li class="fragment"><span>GalNAc</span> is first added by 1 of 20 different <span>ppGALNACT</span> enzymes to the hydroxyl group of Ser or Thr residues. This process builds a basal structure called <span>Tn antigen</span>.</li>
								<li class="fragment">
									Synthesis of core structures is initiated by a single glycosyl-transferase using Tn antigen as the substrate.
								</li>
								<li class="fragment">
									<span>Core1: C1GALT1</span>
								</li>
								<li class="fragment">
									<span>Core3: C3GNT</span>
								</li>
								<li class="fragment">
									These core structures can be further elongated into monoantennary or biantennary structures to make extended glycan chains.
								</li>
							</ul>
						</div>
					</div>
				</section>

				<section>
					<h2>Gut bacteria produce CAZymes that can catalyze glycans</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1.5">
						<img src="img/Review03a.png" alt="review03">
						</div>
						<div class="col" style="flex:1">
							<p>Gut microbes utilizes <span>CAZymes(Carbohydrate-active enzymes)</span> for
								<ul>
									<li>
										Degradaing natural fibre polysaccharides from botanical sources
									</li>
									<li>
										Degrading host mucosal glycans
									</li>
								</ul>
							</p>
							<br>
							<p>Examples</p>
							<ul>
								<li>
									<span>Blood-group-removing GH98 enzymes</span> liberate the terminal trisaccharide from either blood group A or B structures.
								</li>
								<li>
									<span>GH101 enzymes</span> cleave peptide-attached GalNAc but only when it has an attached Tn antigen.
								</li>
								<li>
									<span>Zinc metalloprotease ZmpB from Clostridium perfringens</span> cuts adjacent to glycosylated Ser and/or Thr residues.
								</li>

							</ul>

						</div>
					</div>
				</section>

				<section>
					<h2>Polysaccharide Utilization Loci (PULs)</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1.5">
						<img src="img/Review03b.png" alt="review03">
						</div>
						<div class="col" style="flex:1">
							<p>Gene clusters from the microbes imply that these microbes can degrade O-linked glycans (host mucin)</p><br>
							<ul>
								<li>
									The <span>protease family member from B. thetaiotaomicron (BT4244)</span> prefers a simpler structure with only Tn antigen.
								</li>
								<li>
									Bacteroides caccae possesses 16 M60-like proteases.
								</li>
							</ul>
							<br><br>

							<h5>These mucin-degrading commensal bacteria can cause damage that at least predisposes to disease. <br><br> Also, commensals invading the mucosal barrier further blur the lines between commensal and pathogenic host−microorganism interactions.</h5>
						</div>
					</div>
				</section>

				<section>
					<h2>Pathogens in the small intestine - M cell</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
						<img src="img/Review04a.png" alt="review04a">
						</div>
						<div class="col" style="flex:1">
							<br>
							<h4>M cell: Microfold cell</h4>
							<br>
							<ul class="fragment">
								<li>The major role of M cells is <span>the sampling and transcytosis of antigens</span> through the epithelial barrier.</li>
								<li>M cells cover underlying lymphoid aggregates called <span>Peyer's patches</span>.</li>
							</ul>
							<br>
							<ul class="fragment">
								<li>Certain pathogens use M cells to gain entry through the
									mucosal barrier.
								</li>
								<li>
									Example: <span>Salmonella enterica subsp. enterica serovar Typhimurium </span>
								</li>
							</ul>
						</div>

					</div>
				</section>

				<section>
					<h2>Pathogens in the small intestine - M cell</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
						<img src="img/Review04b.png" alt="review04b">
						</div>
						<div class="col" style="flex:1">
							<br>
							<h5>Some pathogens invade through mucous layer using host cell surface proteins</h5>
							<br>
							<ul>
								<li>
									<span>Yersinia pseudotuberculosis</span> uses its outer membrane protein invasin to transit through M cells into the underlying Peyer’s patch by interacting with <span>β1 integrins</span> expressed on the surface of the targeted cells
								</li>
								<li>
									 <span>Listeria monocytogenes</span> uses two of its main virulence factors, the surface proteins internalin A (InlA) and/or internalin B (InlB), to bind to <span>E-cadherin and MET</span> and invade epithelial cells, including M cells
								</li>
							</ul>
						</div>
					</div>
				</section>

				<section>
					<h2>Pathogens in the small intestine - Tight junction</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
						<img src="img/Review04c.png" alt="review04c">
						</div>
						<div class="col" style="flex:1">
							<h5>Some pathogens secrete enzymes that cleave mucus or disrupt tight junctions in the underlying epithelial cells</h5>
							<br>
							<p>
								<span>Vibrio cholerae</span> secretes a soluble Zn<sup>2+</sup>-dependent metalloproteinase (haemagglutinin protease (Hap)) that possesses mucinolytic activity
							</p>
							<br>
							<p>
								Zonula Occludens Toxin (ZOT) from Vibrio cholerae interacts with the extracellular domains of the tight junction proteins occludin and zonula occludens 1 protein (ZO1) to disrupt epithelial integrity.
							</p>
							<br>
							<p>
								<span>Clostridium perfringens</span> disrupts the tight junction using its enterotoxin targeting <span>claudin</span>.
							</p>
							<br>
							<p>
								A protozoan <span>Giardia lamblia</span> possesses abilities to degrade or disrupt mucus
							</p>
						</div>
					</div>
				</section>

				<section>
					<h2>Pathogens in the small intestine - Goblet cell</h2>
					<div class="container" style="display:flex">
						<div class="col" style="flex:1">
					<img src="img/Review04d.png" alt="review04d">
					</div>
					<div class="col" style="flex:1">
						<br>
						<p><span>L. monocytogenes</span> recognizes E-cadherin at the surface of villi located at the tip of mucus-expelling goblet cells</p>
					</div>
					</div>
				</section>

				<section>
					<h2>Pathogens in the small intestine - Paneth cell</h2>
					<div class="container" style="display:flex">
					<div class="col" style="flex:1">
						<img src="img/Review04e.png" alt="review04e">
					</div>
					<div class="col" style="flex:1">
						<br>
						<p>
							<span>S. Typhimurium</span> has evolved to circumvent host defences during invasion of the epithelial barrier.<br>
							This pathogen can evade the activity of Paneth cell-secreted cationic antimicrobial peptides (CAMPs), including defensins, by altering the anionic
							character of the cell surface.
						</p>
					</div>
					</div>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>The dense commensal microbiota in the large intestine protects the host from invasion by several pathogens</h2>
					<h3>colonization resistance</h3>
					<img src="img/Review05a.png" alt="review05a" width=90%>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Defects in colonic mucous barrier promote invasion of enteric pathogens</h2>
					<img src="img/Review05a.png" alt="review05a" width=80%>
					<p>Mice genetically deficient in mucin 2 (Muc2−/− mice) lack a normal mucous layer, which results in accelerated disease progression following infection with Citrobacter rodentium.</p>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Defects in colonic mucous barrier promote invasion of enteric pathogens</h2>
					<img src="img/Review05a.png" alt="review05a" width=80%>
				<p>Wild-type mice exhibit goblet cell hyperplasia, likely increasing mucus production and expelling the nematode Trichuris muris. However, Muc2−/− mice cannot clear T. muris owing to the lack of mucus production.</p>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Defects in colonic mucous barrier promote invasion of enteric pathogens</h2>
					<img src="img/Review05a.png" alt="review05a" width=80%>
				<p>Mice fed a <span>fibre-free diet</span> exhibit an altered microbiota composition that reflects <span>a higher abundance of mucus-degrading bacteria</span>. Increased foraging on the mucous layer leads to accelerated disease progression following infection with C. rodentium. <br>
				Also, The host mucosal barrier <span>requires the cytokine IL-22</span></p>
				</section>

				<section>
					<h2>Flagella grant an advantage to some pathogens during infection at mucosal surfaces</h2>
					<div class="container" style="display:flex">
					<div class="col" style="flex:1">
					<img src="img/Review05b.png" alt="review05b">
					</div>
					<div class="col" style="flex:1.5">
						<ul>
							<li>Some E. coli strains possess <span>flagella termed H6 and H7, which are adhesive to mucins</span></li>
							<li>
								Similar flagellum-related mucus adherence by <span>Clostridium difficile</span>, a nosocomial pathogen associated with antibiotic treatment, is thought to promote colonization by this organism, which can also form a biofilm near the epithelial surface.
							</li>
						</ul>
					</div>
					</div>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Mucus disruption by pathogens</h2>
					<div class="container" style="display:flex">
					<div class="col" style="flex:1.5">
					<img src="img/Review05c.png" alt="review05c">
					</div>
					<div class="col" style="flex:1">
						<br>
						<ul>
							<li><span>E. coli, Entamoeba histolytica and T. muris</span> produce enzymes and lectins that either cleave or bind to mucins.</li>
							<li>
								StcE, an E. coli metalloprotease, cleaves mucins, which helps the pathogen to reach epithelial cells.
							</li>
							<li>
								T. muris secretes serine proteases that act on the amino-terminal disulfide crosslinking domain of MUC2
							</li>
						</ul>
					</div>
					</div>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Tight junction disruption by pathogens</h2>
					<div class="container" style="display:flex">
					<div class="col" style="flex:1.5">
					<img src="img/Review05c.png" alt="review05c">
					</div>
					<div class="col" style="flex:1">
						<br>
						<ul>
							<li><span>C. difficile and Clostridium perfringens</span> secrete toxins that disrupt colonic tight junctions.</li>
							<li>Outer membrane vesicles (OMVs) filled with proteases are secreted by Campylobacter jejuni and cleave the tight junction proteins E-cadherin and occludin.</li>
							<li>C. difficile toxin A and toxin B disrupt the cellular actin cytoskeleton, which affects its interactions with tight junction components claudin and occludin</li>
						</ul>
					</div>
					</div>
				</section>

				<section>
					<h1>SCFAs and an organic acid can influence pathogens</h1>
					<img src="img/Review06.webp" alt="review05e" width=80%>
				</section>

				<section>
					<!-- Cover for Research paper -->
					<br><br>
					<h6>Research paper</h6>
					<h2> Human papillomavirus and the landscape of secondary genetic alterations in oral cancers</h2>
					<h4>Genome research, 2019</h4>
					<span>2021-35389 Hyunbin Kim / 2021. 11. 16</span>
				</section>

				<section>
					<h2>Abstract</h2>
					<ul>
					<div class="fragment">
					<h6>SAMPLES</h6>
					<li>Genome comparison of 149 HPV-positive and 335 HPV-negative oral squamous cell carcinomas (OSCC) samples (tumor/normal pairs)</li>
					</div>
					<div class="fragment">
					<h6>MUTATIONAL SIGNATURES</h6>
					<li>Different risk factors underlying the two cancer types were reflected in distinctive genomic mutational signatures</li>
					<li>In HPV-positive OSCCs, the signatures of APOBEC cytosine deaminase editing, associated with anti-viral immunity, were strongly linked to overall mutational burden.</li>
					<li>In contrast, in HPV-negative OSCCs, T>C substitutions in the sequence con- text 5′-ATN-3′ correlated with tobacco exposure.</li>
					</div>
					<div class="fragment">
					<h6>GENES WITH MUTATIONS</h6>
					<li>Significant enrichment of somatic mutations was confirmed or newly identified in PIK3CA, KMT2D, FGFR3, FBXW7, DDX3X, PTEN, TRAF3, RB1, CYLD, RIPK4, ZNF750, EP300, CASZ1, TAF5, RBL1, IFNGR1, and NFKBIA.</li>
					<li>
					High-ranking variant allele fractions implicated ZNF750, PIK3CA, and EP300 mutations as candidate driver events in HPV-positive cancers.
					</li>
					</div>
					<div class="fragment">
					<h6>GENE EXPRESSION & CNV</h6>
					<li>Universal expression of HPV E6 and E7 oncogenes was a sine qua non of HPV-positive OSCCs</li>
					<li>Frequent copy number changes were associated with concordant changes in gene expression. Chr 11q (including CCND1) and 14q (including DICER1 and AKT1) were recurrently lost in HPV-positive OSCCs, in contrast to their gains in HPV-negative OSCCs</li>
					</div>
					</ul>
				</section>

				<section>
					<img src="img/HeadandNeck.png" alt="" width=100%>
				</section>

				<section>
					<h2>Mechanism of HPV infection</h2>
					<div class="container" style="display:flex">
					<div class="col" style="flex:1.5">
					<img src="img/HPV01.png" alt="">
					</div>
					<div class="col" style="flex:1">
					<p>Mechanism of HPV infection affecting the occurrence and development of cancer.</p>
					<p>After HPV infects mucosal epithelial cells, its DNA is integrated into the host cell genome, thereby resulting in <span>overexpression of E6 and E7 mRNA</span> and altered function of various proteins in host cells. </p>
					<p>E6 and E7 products <span>degrades (or inactivates) pRb and P53 which act as tumor suppressor</span> </p>
					</div>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Pipeline</h2>
					<img src="img/Pipeline.png" alt="">
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Pipeline</h2>
					<div class="container" style="display:flex">
					<div class="col" style="flex:1">
					<img src="img/Pipeline.png" alt="">
					<ul class="fragment">
						<li>HPV transcript alignment - GSNAP, Cufflinks</li>
						<li>SNV detection - MuTect, Strelka (SNV)</li>
						<li>CNV calling - In house script</li>
						<li>Mutational signature - deconstructSigs, 30 pre-defined COSMIC cancer mutation signatures</li>
						<li>VAF analysis - MuTect, Strelka, LoFreq, Samtools</li>
					</ul>
					</div>
					<div class="col" style="flex:1">
					<img src="img/SNV_CNV.png" alt="">
					</div>
				</section>

				<section>
					<h2>Recurrent mutations in HPV+ oral cancers</h2>
					<img src="img/Research01.png" alt="" width=80%>
					<p>Recurrent mutations in PIK3CA, KMT2D, FGFR3, FBXW7, DDX3X, PTEN, TRAF3, RB1, CYLD, RIPK4, ZNF750, EP300, CASZ1, TAF5, RBL1, IFNGR1, and NFKBIA</p>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Most significantly mutated genes:</h2><h2>HPV+ oral cancers vs HPV- oral cancers</h2>
					<img src="img/Research04.png" alt="" width=50%>
					<ul>
						<li>HPV+ oral cancers: PIK3CA, ZNF750, FGFR3, CASZ1, PTEN, CYLD, DDX3X</li>
						<li>HPV- oral cancers: TP53, FAT1, CDKN2A, NOTCH1, CASP8, HRAS</li>
					</ul>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Most significantly mutated genes:</h2><h2>HPV+ oral cancers vs HPV+ cervical cancers</h2>
					<img src="img/Cervical.png" alt="" width=50%>
					<ul>
						<li>HPV+ oral cancers: PIK3CA, ZNF750, FGFR3, CASZ1, PTEN, CYLD, DDX3X</li>
						<li>HPV+ cervical cancers: PIK3CA, EP300, FBXW7, PTEN</li>
					</ul>
				</section>

				<section>
					<div class="container" style="display:flex">
					<div class="col" style="flex:2">
					<img src="img/Research05.png" alt="">
					</div>
					<div class="col" style="flex:1">
						<br>
						<h3>Somatic mutations alter protein structures and functions.</h3>
					</div>
					</div>
				</section>

				<section>
					<h2>Mutational signature analysis</h2>
					<img src="img/Mutsig01.png" alt="" width=60%>
					<p>By analyzing the context of the variants, we can find out why these variants are accumulated in the genome</p>
				</section>

				<section>
					<img src="img/Mutsig02.png" alt="" width=100%>
				</section>
				<section>
					<img src="img/Mutsig03.png" alt="" width=100%>
				</section>
				<section>
					<br>
					<h2>How signature is generated</h2>
					<img src="img/Mutsig04.png" alt="" width=100%>
				</section>

				<section>
					<img src="img/Mutsig05.png" alt="" width=100%>
				</section>
				<section>
					<img src="img/Mutsig06.png" alt="" width=100%>
				</section>

				<section>
					<br>
					<h2>Signatures related to oral cancers</h2>
					<img src="img/Mutsig07.png" alt="" width=100%>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Signature analysis result - HPV+ oral cancers</h2>
					<img src="img/Research02.png" alt="">
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Signature analysis result - HPV+ oral cancers</h2>
					<div class="container" style="display:flex">
					<div class="col" style="flex:2">

					<img src="img/Research02.png" alt="" width=100%>
					<img src="img/APOBEC01.jpg" alt="" width=100%>
					<img src="img/APOBEC02.jpg" alt="" width=100%>
					</div>
					<div class="col" style="flex:3">
						<br>
					<p> Most pronounced among HPV-positive OSCCs were the fractions of C>T and C>G mutations observed in the context of 5′ -T∗CW-3′ genomic sequences associated with APOBEC cytidine deaminase editing</p>
					</div>
				</section>

				<section>
					<img src="img/Mutsig08.png" alt="" width=80%>
				</section>

				<section>
					<img src="img/Mutsig09.png" alt="" width=80%>
				</section>

				<section>
					<div class="container" style="display:flex">
					<div class="col" style="flex:0.9">
					<br>
					<br>
					<img src="img/VAF02.png" alt="">
					</div>
					<div class="col" style="flex:1">
					<img src="img/VAF01.png" alt="">
					</div>
				</section>

				<section>
					<h2>Driver mutation predicted with VAF</h2>
					<img src="img/Research08.png" alt="" width=80%>
				</section>

				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Copy number variation</h2>
					<img src="img/Research06.png" alt="" width=70%>
					<ul>
					<li>HPV+: 3q gain, 11q loss</li>
					<li>HPV-: 8q, 14 gain, 18q loss</li>
					</ul>
				</section>
				<section data-auto-animate data-auto-animate-unmatched="fade">
					<h2>Copy number variation</h2>
					<img src="img/Research07.png" alt="">
					<ul>
					<li>HPV+: 3q gain, 11q loss</li>
					<li>HPV-: 8q, 14 gain, 18q loss</li>
					</ul>
				</section>

				<section>
					<h2>Summary</h2>
					<img src="img/Research09.png" alt="" style="display:block;margin-left:auto;margin-right:auto;" width=50%>
				</section>
				<section>
					<h2>Additionaly,</h2>
					<img src="img/ecoli.jpeg" alt="" width=80%>
					<h6>https://www.vanboxtellab.eu/publications/mutational-signature-in-colorectal-cancer-caused-by-genotoxic-pks-e-coli</h6>
				</section>

			</div>
		</div>

		<script src="dist/reveal.js"></script>
		<script src="plugin/notes/notes.js"></script>
		<script src="plugin/markdown/markdown.js"></script>
		<script src="plugin/highlight/highlight.js"></script>
		<script>
			// More info about initialization & config:
			// - https://revealjs.com/initialization/
			// - https://revealjs.com/config/
			Reveal.initialize({
				hash: true,
				width: 1600, height: 900,
				// Learn about plugins: https://revealjs.com/plugins/
				plugins: [ RevealMarkdown, RevealHighlight, RevealNotes ]
			});
		</script>
	</body>
</html>
